packages/suite/src/views/wallet/send/components/Outputs/components/Amount/index.tsx

```diff
@@ -90,6 +90,9 @@ const Amount = ({ outputId }: { outputId: number }) => {
         composeTransaction,
     } = useSendFormContext();
 
+    // TODO: There is no guarantee that using arbitrary integer as an index won't result in undefined
+    if (!outputs[outputId]) return null;
```

Comment:
interesting, is it caused by this animation? looks like a bug

Reply:
my theory is that there is some delay thanks to animation before the Output component is removed from the DOM, so it technically still renders Amount component with index of an output that was already removed from reducer.

anyway imo it it still wrong that outputs[outputId] is not typed with potential `undefined`, if I pass out-of-range outputId TS will stay silent and app throws runtime error.